config/credentials.yml.enc

```diff
@@ -1 +1 @@
-E4HE9d9aaDsjXgc0WnjWiTR9hYzSwtFefYObqO3ySK+IM4M0dR3jUZVyQ29dzKEKtHf4z6z4mCAfsI3/IBP8ixJzeumDspsnvxlEtO8sxqmlpYZP80Bm3wzkifKCUhGFEwTvGZqzOlAzzqQaWF70Y3PVIkNRnBAfo29TzG7bBStwfUYOXhJKIbUqjsHMX9dqKeYrjWnxhQrBjIxjMVlQOPHkVqSnv/mGBgf+bs1qOZ6hR4SG7eOxFbHQxQyfZezRRgi1fHdP/C3Fc7eiPvqiSvon8LoBdpht6xYAauQvP1tEkSHvjRJHjhtybFe6NFe1zk++YSgr/FmEkvIVRkFB+OhjTefoDemDriRnRHd73FyYaRWDPQlc+Lnk+Gy2kaeMgIn8n+W/9+pRxi1jz6PF72Yms21KAhbvfcyR--x+ReK0Qv71uCx3/Y--AGHhdfGLe0+C5DdEnKRM0w==
\ No newline at end of file
+RNhdDFxSD/w76HAIA8zo7kI/nwdtW+E1Xue/ALK9LzGICzCVhHsIDbgpZdVKnOzIHb2mLXTSM/WLlrudqwdR7ZqOlHFvSyz13J5MKGPM7cjzaCjtcwhlXZQWpUVKwabYOhTTeObBvTYLMP9HI6HKt6URvjnMTGa/WCLD9jUc5+YbPNEy1Vp8NfGYOZAOz5TWq8Qwx7LANCZ9DT71+UJQmEj2L8elMSA0aUSOr5mWyvrw08+fTj7OaSuW9Agp64xZUamblvh/RWDxbDsCfY2lN4iuV+noB0kW2pvvezx3BCbKYrXra4Fo+Htkg4zrYZYUr19yIU2jsM4qUtdypgUmRzspVEXmHco2r9DKP14+QMwnZWxffLqxjvvT+aSAElyHosQYg/5Lt6824LkdOWWlD1ecflOgYuO8gGwMa9WccA4ci+72zy5j94QG/abb0ywtyB8ZIuwoFnJ2n2GIVfqGp3s1G0bVzPgJitXy8B3wcP2p+Y2kb/cGIjYiNHnxyjrEM1bIaGZcFelPPrCfb5k32D5EW/OX18es39v/nfl7ayoUBj7sebbAqmAwHQHwJzX+Hk1RXFb/--GDPLgKkrE4hkyb+J--F5Mx3iNqzwz9d4LbwLlkOw==
```

Comment:
Did you want to commit this? I saw you added it to your `.gitignore` which makes me think not. Because the file is already in git you'll to tell git to remove it. I think you'll need `git rm --cached config/credentials.yml.enc` but it's confusing and I never remember how it works.

Here's some more info: https://stackoverflow.com/questions/13541615/how-to-remove-files-that-are-listed-in-the-gitignore-but-still-on-the-repositor

Docs: https://git-scm.com/docs/git-rm#Documentation/git-rm.txt---cached

Reply:
Ooh yes! I also forget how to do it after not doing it for awhile. I will bookmark those links. 👍

Reply:
Soo.. after further investigation, I found a site [here](https://blog.eq8.eu/til/rails-52-credentials-tricks.html) that says: 

> it looks like It is ok to commit `config/credentials.yml.enc` to git (that is its purpose)

Never commit config/master.key to git!
Never commit value of RAILS_MASTER_KEY to git!
>

Reply:
Sounds good to me!